### DISCUSSION TITLE
state of the package?

### DISCUSSION THREAD

**Discussion by DiTo97** [bluet/proxybroker2#146] [General]
hi @bluet,

I was wondering what the state of the package is like.

I had been working on a proxy broker of mine for python 3.10 (a handful of sources and HTTPS only), since I could not find anything that I liked and felt was easy enough to work with, until I came across your revamp of this 5-year-old package.

after browsing through the issues, I found out that you recently moved to poetry as package manager (which I agree with), but it is not clear to me what is the roadmap ahead in terms of development, documentation and release on PyPI.

the latter seems to be pending until your transition to python 3.10 is complete, am I right?

also, the README seems to be a copy-paste of the old fork, still pointing to the old package release on PyPI and with outdated TODOs.

tl;dr - I don't think it would make much sense for me to keep developing a small package with little maintenance; however, the package was being written for python 3.10, so I should be able to help in transitioning the async code to that version.

**Comment by DiTo97** [comment]
are you still active, @bluet ?

the repository I mentioned is [here](https://github.com/DiTo97/proxy-rotation), but I think we should join forces.
